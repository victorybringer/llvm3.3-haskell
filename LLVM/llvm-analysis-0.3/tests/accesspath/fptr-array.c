#include <stdlib.h>

struct Vtbl {
  void* (*malloc_ptr)(size_t);
  void (*free_ptr)(void*);
  void* (*realloc_ptr)(void*, size_t);
  void* (*calloc_ptr)(size_t, size_t);
};

void f(struct Vtbl *vtbl) {
  vtbl->calloc_ptr = calloc;
}
