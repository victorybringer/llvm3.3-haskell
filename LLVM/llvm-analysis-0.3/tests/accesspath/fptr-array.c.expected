fromList [ ("@f", ("%struct.Vtbl*", [AccessDeref, AccessField 3])) ]
